src/borg/testsuite/archiver.py

```diff
@@ -3603,6 +3603,39 @@ def test_recovery_from_deleted_repo_nonce(self):
         self.cmd('create', self.repository_location + '::test2', 'input')
         assert os.path.exists(nonce)
 
+    def test_init_defaults_to_argon2(self):
```

Comment:
hmm, we "borg init" quite a lot in `borg.testsuite.archiver`.

If we slow that down via very secure argon2 parameters, our tests will run much slower.
guess we should have some way to accelerate that and using different params for testing and real world.

Reply:
I am working on a solution for this. The first step is to measure the potential savings.

Reply:
that problem isn't new btw. - maybe pbkdf2 also slowed down our tests already before your PR, not sure how much.

so a good comparison would be the default kdf against some  tuned-for-tests-only kdf.

Reply:
Look what I found:
https://github.com/borgbackup/borg/blob/22fc6d1bddc91c1bea163d906ce40f85c8744655/conftest.py#L8-L10

Reply:
I have performed an experiment: we can expect savings of about 220 +-20 seconds in a full test run. (note: I skipped test files that do not call the KDF because I was worried my benchmark would exceed github max timeout)

https://github.com/hexagonrecursion/borg/runs/5804738142?check_suite_focus=true

- [ ] TODO: patch conftest.py

Reply:
definitely worth saving that time / resources.